### PR TITLE
Handle static guim domain images

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Handle static domain for images

## Why?

Some older articles are not displaying the proper carousel images

## Screenshots

| Before      | After      |
|-------------|------------|
| ![before][] | ![after][] |

[before]: https://user-images.githubusercontent.com/76776/197539098-f92c8622-78da-4a36-a4e9-091ecb78ba52.png
[after]: https://user-images.githubusercontent.com/76776/197539032-a2197157-1018-4935-9c4a-33e69576e09f.png

### CODE REVIEW

File: dotcom-rendering/src/web/components/Carousel.importable.tsx [hunk 1]
```diff
@@ -437,14 +437,23 @@ const HeaderAndNav: React.FC<HeaderAndNavProps> = ({
...
+
+	if (url.hostname === 'i.guim.co.uk' && prefix) {
+		switch (prefix) {
+			case '/img/media/':
+				return `https://media.guim.co.uk/${url.pathname.replace(
+					prefix,
+					'',
+				)}`;
+			case '/img/static/':
+				return `https://static.guim.co.uk/${url.pathname.replace(
```
Comment (OllysCoding) [inline dotcom-rendering/src/web/components/Carousel.importable.tsx:452]: Why are we changing the hostname here? I'm guessing if we do then fastly resizer wouldn't do anything?

Comment (mxdvl) [reply dotcom-rendering/src/web/components/Carousel.importable.tsx:452]: This is where the master files are hosted. It’s being turned back into a `i.guim` domain by `Picture.tsx`:

https://github.com/guardian/dotcom-rendering/blob/6e143dbc2a63d0c9d884427f1b47d737d6c00cde/dotcom-rendering/src/web/components/Picture.tsx#L180-L181

This whole approach is a temporary reverse-engineering of the signing process.

**Comment by paperboyo** [comment]
Just wanted to say this domain has a completely different structure to `media` (and no `master`), eg.
`i.guim.co.uk/img/static/sys-images/Guardian/Pix/pictures/2007/12/20/omagh372.jpg`

We also have these `sport` and `uploads` ones… (eg. `port.guim.co.uk/football/crests/120/7497.png` & `i.guim.co.uk/img/uploads/2018/06/04/Kate_Connolly,_L.png`)

File: dotcom-rendering/src/web/components/Carousel.importable.tsx (hunk 1 above)
Comment (OllysCoding) [reply dotcom-rendering/src/web/components/Carousel.importable.tsx:452]: Ah okay ... definitely weird, we should get the full solution done asap then!

**Review by OllysCoding — APPROVED** [review]
+1 confusing, but this works, thanks Max! :)

**Pull request merged**